product_abc_classification_sale_stock/tests/test_abc_classification_profile.py

```diff
@@ -6,7 +6,7 @@
 from odoo.tests.common import TransactionCase
 
 
-class TestABCClassificationProfile(TransactionCase):
+class TestABCClassificationProfileSaleStock(TransactionCase):
```

Comment:
@chaule97 This change is useless but if you change the classname you must change the filename.

Reply:
I have renamed

Reply:
okay, I have removed this commit. I remember Odoo 17 have issue when it have duplicate testcase class name. I think that it was fixed in version 18.0

https://github.com/OCA/pos/pull/1232